### PR TITLE
feat(ephemeral-resource): Create votingapp ecr repo

### PR DESCRIPTION
**Pull request by leohauschild**

### CODE REVIEW

**Review by benhur-araujo — APPROVED** [review]

**Comment by carlosgit2016** [comment]
I think the conquer-ecr-repo can be deleted. Why are keeping it ? https://github.com/conquer-project/conquer_platform/pull/15/files#diff-dd0379c7ab0c42a28fe40e6ea61c883e0def8e0a3d2b168da9886bcc521f230bR1

**Comment by carlosgit2016** [comment]
Please wait for Benhur PR to be merged to avoid destroying resources https://github.com/conquer-project/conquer_platform/pull/15#issuecomment-1784164940

File: infrastructure/ecr.tf [hunk 1]
```diff
@@ -7,3 +7,13 @@ resource "aws_ecr_repository" "conquer-ecr-repo" {
     scan_on_push = true
   }
 }
+
+resource "aws_ecr_repository" "votingapp-ecr-repo" {
+  name                 = "votingapp"
```
Comment (carlosgit2016) [inline infrastructure/ecr.tf:12]: didn't we agreed on one per microservice ?

```terraform
locals {
  microservices = [
    "votingapp-results"
  ]
}

resource "aws_ecr_repository" "ms" {
  for_each = toset(local.microservices)
  name = each.value
}
```

Comment (leohauschild) [reply infrastructure/ecr.tf:12]: we talked about creating a registry per application, and separating microservices by folders

**Pull request merged**